### PR TITLE
Changed red,green,blue to r,g,b in RGB(a) Colors

### PR DESCRIPTION
**Pull request by marcusnaslund**
First part of solving #1412, rename `red`, `green`, `blue` to `r`, `g`, `b`. The former names will be used as static members in a later PR.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @davidpiuva or @emilwestergren ?

**Comment by davidpiuva** [comment]
ok

File: source/draw/Color.ooc [hunk 1]
```diff
@@ -82,38 +82,38 @@ ColorYuva: cover {
...
 	}
 	operator == (other: This) -> Bool { this equals(other) }
 	operator != (other: This) -> Bool { !this equals(other) }
 }
 
 ColorRgba: cover {
-	red, green, blue, alpha: Byte
-	normalized ::= FloatTuple4 new(this red as Float / 255, this green as Float / 255, this blue as Float / 255, this alpha as Float / 255)
-	init: func@ (=red, =green, =blue, =alpha)
+	r, g, b, alpha: Byte
```
Comment (emilwestergren) [inline source/draw/Color.ooc]: What about alpha?

Comment (marcusnaslund) [inline source/draw/Color.ooc]: Ah, I was just about to ask that. You decide.

Comment (emilwestergren) [inline source/draw/Color.ooc]: I think `a` for consistency

**Comment by davidpiuva** [comment]
Looks good to me. @emilwestergren can set the flag to `final review` once done.

File: source/draw/Color.ooc (hunk 1 above)
Comment (marcusnaslund) [inline source/draw/Color.ooc]: Coming up.

**Comment by marcusnaslund** [comment]
Updated @emilwestergren

**Pull request merged**